django/db/models/indexes.py

```diff
@@ -44,22 +44,27 @@ def check_name(self):
             self.name = 'D%s' % self.name[1:]
         return errors
 
-    def create_sql(self, model, schema_editor, using=''):
+    def get_sql_create_template_values(self, model, schema_editor, using):
         fields = [model._meta.get_field(field_name) for field_name, order in self.fields_orders]
         tablespace_sql = schema_editor._get_index_tablespace_sql(model, fields)
         quote_name = schema_editor.quote_name
         columns = [
             ('%s %s' % (quote_name(field.column), order)).strip()
             for field, (field_name, order) in zip(fields, self.fields_orders)
         ]
-        return schema_editor.sql_create_index % {
+        return {
             'table': quote_name(model._meta.db_table),
             'name': quote_name(self.name),
             'columns': ', '.join(columns),
             'using': using,
             'extra': tablespace_sql,
         }
 
+    def create_sql(self, model, schema_editor, using='', parameters=None):
```

Comment:
I don't see the need for the `parameters=None` addition. Is it intentionally unused for now?